test/Serialization/load-target-fallback.swift

```diff
@@ -0,0 +1,24 @@
+// RUN: %empty-directory(%t)
+// RUN: mkdir %t/TargetLibrary.swiftmodule
+// RUN: %target-swift-frontend -emit-module -o %t/TargetLibrary.swiftmodule/%module-target-triple.swiftmodule %S/Inputs/def_func.swift -module-name TargetLibrary
+// RUN: touch %t/TargetLibrary.swiftmodule/%target-cpu.swiftmodule
+
```

Comment:
I feel like a comment describing what this test does would be helpful to future maintainers.